testsuite/integration-arquillian/servers/app-server/jboss/eap/src/main/java/org/keycloak/testsuite/arquillian/eap/container/EAPAppServerProvider.java

```diff
@@ -40,6 +40,7 @@
     private final String startupTimeoutInSeconds;
 
     public EAPAppServerProvider() {
+        containerName = System.getProperty("app.server");
```

Comment:
Wouldn't be better to also test that property is not null or empty? I know it is set by property in pom.xml but in case someone accidentally removes it it will start failing with NPE. See the lines below.

Reply:
@vramik Sorry I missed this validation. Added: Validate.notNullOrEmpty(containerName, "app.server is not set.");